tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -0,0 +1,507 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.Data;
+using Xunit;
+
+namespace Microsoft.Data.Tests
+{
+    public class DataFrameTests
+    {
+        static public DataFrame MakeDataFrameWithTwoColumns(int length)
```

Comment:
(nit) the accessibility comes first

`public static ...`

See number 5 here: [https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

> We always specify the visibility, even if it's the default (e.g. private string _foo not string _foo). Visibility should be the first modifier (e.g. public abstract not abstract public).